Dockerfile_openvino_base

```diff
@@ -0,0 +1,22 @@
+FROM openvino/ubuntu18_runtime
+USER root
+RUN apt-get update && apt-get install -y python3-dev
+RUN pip3 install virtualenv
+WORKDIR /ie-serving-py
+RUN chown openvino .
+USER openvino
+COPY requirements.txt /ie-serving-py/
+RUN virtualenv -p python3 .venv && \
```

Comment:
Maybe we should start using `python3 -m venv`? It has better support, its built into python3 in most distros (not Ubuntu, still need to install `python3-vevn`) and it's suggested to use with Python3.3+.